Automattic/wp-calypso
# SMP: refactor mshot hook to use response status instead of loading image dimensions

#### Proposed Changes

* Define the mshot isLoading state by fetching the status and image headers instead of using the image resolutions.

Image resolutions was good enough until we added responsive images.

#### Testing Instructions

* Go to `/site`
* Filter by public sites
* Select grid to view the site thumbnails bigger with responsive size.
* Observe the thumbnails are loaded correctly
* Reduce the window size to try the responsive images
* Refresh the page
* Observe the thumbnails are loaded correctly
* Open a new tab, and create a new post on a simple site
* Go back to the previous tab `/sites`  and check the thumbnail is updated.

#### Pre-merge Checklist






Related to #
